drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1978,6 +1988,14 @@ void RasterizerSceneGLES3::render_scene(const Ref<RenderSceneBuffers> &p_render_
 		_draw_sky(render_data.environment, render_data.cam_projection, render_data.cam_transform, sky_energy_multiplier);
 	}
 
+	if (rt->copy_fbo > 0) {
```

Comment:
This logic should only run if needed. It should not run every frame as that will needlessly hurt performance for users who dont use the screen or depth textures